.github/workflows/e2e_aws.yaml

```diff
@@ -94,16 +100,19 @@ jobs:
         uses: actions/setup-go@44694675825211faa026b3c33043df3e48a5fa00 # v6.0.0
         with:
           go-version: ${{ env.GO_VERSION }}
+          cache-dependency-path: "**/go.sum"
 
       - uses: oras-project/setup-oras@22ce207df3b08e061f537244349aac6ae1d214f6 # v1.2.4
         with:
           version: ${{ env.ORAS_VERSION }}
 
       - name: Extract qcow2 from ${{ inputs.podvm_image }}
         if: ${{ !inputs.oras }}
+        env:
+          PODVM_IMAGE: "${{ inputs.podvm_image }}"
         run: |
-           qcow2=$(echo "${{ inputs.podvm_image }}" | sed -e "s#.*/\(.*\):.*#\1.qcow2#")
-           ./hack/download-image.sh "${{ inputs.podvm_image }}" . -o "${qcow2}"
+           qcow2=$(echo "${PODVM_IMAGE}" | sed -e "s#.*/\(.*\):.*#\1.qcow2#")
```

Comment:
I think this the line that shell check is failing on with: https://www.shellcheck.net/wiki/SC2001. It is interesting that it didn't complain previously though.

Don't ask me how look it took to work this out, but I think `qcow2=$(tagless=${PODVM_IMAGE%%:*}; echo ${tagless##*/})` might to what you are trying to do?